internal/hmyapi/apiv2/sendtxargs.go

```diff
@@ -33,14 +33,14 @@ func (args *SendTxArgs) setDefaults(ctx context.Context, b Backend) error {
 		*(*uint64)(args.Gas) = 90000
 	}
 	// TODO(ricl): add check for shardID
-	if args.GasPrice == nil {
-		// TODO(ricl): port
```

Comment:
why this whitespace change

Reply:
Since there is no content in the if statement and reported by static check.

Reply:
Ah but you comment it out , then just delete , don’t comment out